corehq/apps/domain/views.py

```diff
@@ -3271,6 +3276,10 @@ def exception_safe_password_reset(request, *args, **kwargs):
     # http://streamhacker.com/2009/09/19/django-ia-auth-password-reset/
     # http://www.rkblog.rk.edu.pl/w/p/password-reset-django-10/
     # http://blog.montylounge.com/2009/jul/12/django-forgot-password/
+    kwargs['password_reset_form'] = partial(HQPasswordResetForm, domain)
+    if domain:
+        kwargs['extra_context']['domain_name'] = domain
+        kwargs['extra_email_context'] = {'domain_name': domain}
     try:
         return password_reset(request, *args, **kwargs)
     except None:
```

Comment:
Weird. What does `except None` do? (I know you didn't write this, just never seen it before).